source/postprocess/composition_velocity_statistics.cc

```diff
@@ -86,8 +86,9 @@ namespace aspect
       double velocity_square_integral_selected_fields = 0., area_integral_selected_fields = 0.;
       for (unsigned int c = 0; c < this->n_compositional_fields(); ++c)
         {
-          vrms_per_composition[c] = std::sqrt(global_velocity_square_integral[c]) /
-                                    std::sqrt(global_area_integral[c]);
+          if (global_area_integral[c] > 0)
+            vrms_per_composition[c] = std::sqrt(global_velocity_square_integral[c]) /
+                                      std::sqrt(global_area_integral[c]);
```

Comment:
This kind of change makes me look around what happens/is supposed to happen if `global_are_integral[c] > 0` is not true. Here, that would imply that `vrms_per_composition[c]` remains zero (that's the default initialization, but that's not explicit anywhere in the code), which may or may not be what we want. I think in the current context that's fine -- we just print zero to screen for these components.

In general, though, I think it would make the code easier to read if we had an explicit `else` branch. It would have saved me a couple of minutes of scrolling around. I'll tackle this in a follow-up patch.

Reply:
Yes, I thought about this as well when I made this change: If there's a compositional field whose value is below 0.5 throughout the whole domain, we want to write some kind of output. Zero is one option, but outputting an invalid value could make sense as well to distinguish it from other fields that have a nonzero area/volume in models where the velocity just happens to zero.

Reply:
Good point, too. In https://github.com/geodynamics/aspect/pull/6268, I left it at zero. But we could also output NaN. Any preference? I'd be happy to write a follow-up to output NaN -- perhaps that's a better choice anyway, to indicate that the value could simply not be computed, rather than that we computed a zero.